File: notifications/register/create-binding-server-apns/create-binding-server-apn.6.x.py
#!/usr/bin/env python
# Install the Python helper library from twilio.com/docs/python/install
import os

from twilio.rest import Client

# To set up environmental variables, see http://twil.io/secure
ACCOUNT_SID = os.environ['TWILIO_ACCOUNT_SID']
AUTH_TOKEN = os.environ['TWILIO_AUTH_TOKEN']

client = Client(ACCOUNT_SID, AUTH_TOKEN)
binding = client.notify.services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX') \
    .bindings.create(endpoint='endpoint_id',
                     identity='00000001',
                     binding_type='apn',
                     address='apn_device_token',
                     tag=['preferred device', 'new user'])

print(binding.sid)

File: notifications/register/create-binding-server/create-binding-server-apn.6.x.rb
# Download the Ruby helper library from twilio.com/docs/libraries/ruby
require 'twilio-ruby'

# Get your Account Sid and Auth Token from https://www.twilio.com/console
# To set up environmental variables, see http://twil.io/secure
account_sid = ENV['TWILIO_ACCOUNT_SID']
auth_token = ENV['TWILIO_AUTH_TOKEN']

client = Twilio::REST::Client.new(account_sid, auth_token)

service = client.notify.v1.services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')

binding = service.bindings.create(
  identity: '00000001',
  endpoint: 'endpoint_id',
  binding_type: 'apn',
  address: 'apn_device_token',
  tag: 'preferred device, new user'
)

puts binding.sid
